samples/v1.6/Elements/CompoundButton.json

```diff
@@ -106,12 +99,10 @@
             "title": "Pending Taks",
             "icon": {
                 "name": "Emoji",
-                "size": "Medium",
-                "style": "Regular",
-                "color": "attention"
+                "style": "Regular"
             },
             
-            "badge": "priority",
+            "badge": "Priority",
```

Comment:
I hope this String works in Desktop client as well. We can't have different string for mobile.

Reply:
sure, which string specifically you are talking about, the compoundButton.json works for desktop client as well to produce adaptive cards

Reply:
We changed from "priority" to "Priority".  It works as expected in all clients, right?